include/libu64/stackcheck.h

```diff
@@ -3,20 +3,24 @@
 
 #include "ultra64.h"
 
+typedef enum StackStatus {
+    /* 0 */ STACK_STATUS_OK,
+    /* 1 */ STACK_STATUS_WARNING,
+    /* 2 */ STACK_STATUS_OVERFLOW
+} StackStatus;
+
 typedef struct StackEntry {
     /* 0x00 */ struct StackEntry* next;
     /* 0x04 */ struct StackEntry* prev;
-    /* 0x08 */ u32 head;
-    /* 0x0C */ u32 tail;
+    /* 0x08 */ u32* head;
+    /* 0x0C */ u32* tail;
```

Comment:
Those are pointers to the head and tail of the stack. I think `void*` would be more correct for this.
As a reference MM uses `void*` for this